app/controllers/claims_controller.rb

```diff
@@ -164,4 +167,51 @@ def add_answers_to_rollbar_context
       )
     end
   end
+
+  def check_whether_closed_for_submissions
+    return if session[:submitted_claim_id].present?
```

Comment:
Do we need this guard? Seems unlikely that someone would submit a claim right as the journey is closed